### PR TITLE
Merge branch 'add-coordinator-book-appointment-function' into add-cor…

### PR DESCRIPTION
**Pull request by yedidya03**
…dinator-book-non-manual

### CODE REVIEW

File: packages/functions/src/coordinator/CoordinatorBookAppointmentHandler.ts [hunk 1]
```diff
@@ -7,13 +7,17 @@ export default async function (
   callerId: string
 ): Promise<FunctionsApi.BookAppointmentResponse> {
   if (request.donorId !== MANUAL_DONOR_ID) {
-    throw Error(`Coordinator booking for real donots is not supported yet`);
-  }
+    const donors = await CoordinatorDAL.getAdminDonors(callerId);
```
Comment (yaron1m) [inline packages/functions/src/coordinator/CoordinatorBookAppointmentHandler.ts:10]: This is not very efficient. 
Why do you need to fetch ALL the coordinator donors, if you already know the donor id?

Comment (yedidya03) [reply packages/functions/src/coordinator/CoordinatorBookAppointmentHandler.ts:10]: it is mainly for group coordinators that should be able to book only their donors..

Comment (yedidya03) [reply packages/functions/src/coordinator/CoordinatorBookAppointmentHandler.ts:10]: however, what we can do is check the other coordinator types before doing that..

Comment (yaron1m) [reply packages/functions/src/coordinator/CoordinatorBookAppointmentHandler.ts:10]: Let's say this is a group coordinator.

Why do you need to fetch all their donors? Why not just he one you want to validate?

**Comment by yaron1m** [comment]
Any update here/

**Pull request closed without merging**